doc/events.md

```diff
@@ -1,9 +1,11 @@
 # Events
 
 _This functionality is available in versions 0.3.0 and up._
+Note that prior to version 1.0.0 the Google Guava EventBus (`com.google.common.eventbus` package)
+API was used, but now there is an API provided in the `com.cloudant.sync.event` package.
```

Comment:
something like "to migrate your code you will need to adjust your imports occasionally"